CaImAn/caiman_main.py

```diff
@@ -46,6 +47,16 @@ def main(caiman_parameters):
 
     Yr, dims, T, cnm, images = cnmf(n_processes, dview, fname_new, caiman_parameters)
 
+    # CaimAn Cross register
+    A = cnm.estimates.A[:,cnm.estimates.idx_components]
+    C = cnm.estimates.C[cnm.estimates.idx_components,:]
+    AC = A.dot(C)
+    shape = (dims[0],dims[1],T)
+    AC = AC.reshape(shape, order='F').transpose((-1, 0, 1))
+    A = A.toarray()
+    A = A.reshape(shape[0],shape[1],A.shape[1], order='F').transpose((-1, 0, 1))
+    A = cross_register(A, AC, caiman_parameters)
```

Comment:
in line 78 it is saving `cm.estimates.A[...]` so I'm not sure `A = cross_register(A, AC, caiman_parameters)` will work